src/components/notices-control-view/Notice.vue

```diff
@@ -0,0 +1,218 @@
+<template>
+  <div class="notice">
+    <label>Заголовок</label>
+    <input
+      :disabled="!edit"
+      class="notice__title"
+      :class="{ notice__title__background: !title }"
+      v-model="titleModel"
+    />
+    <label>Подробности</label>
+    <textarea
+      :disabled="!edit"
+      class="notice__description"
+      :class="{ notice__description__background: !description }"
+      v-model="descriptionModel"
+    />
+    <div class="notice__checkbox-container">
+      <div class="notice__authorized-label">Видно только волонтерам:</div>
+      <a-switch v-model:checked="authorizedModel" :disabled="!edit" />
+    </div>
+    <div class="notice__btn-container">
+      <Button v-if="!edit" @click="onEdit" class="notice__edit-btn" title="редактировать" />
+      <Button v-if="edit" @click="onUpdate" class="notice__update-btn" title="сохранить" />
+      <Button v-if="edit" @click="onCancel" class="notice__cancel-btn" title="отменить" />
+      <Button @click="onDelete" class="notice__delete-btn" title="удалить" />
+    </div>
+  </div>
+</template>
+
+<script>
+import { defineComponent, ref } from 'vue';
+
+import Button from '@/components/common/Button.vue';
+
+export default defineComponent({
+  name: 'Notice',
+  props: {
+    noticeId: String,
+    title: String,
+    description: String,
+    authorized: Boolean,
+  },
+  components: { Button },
+  created() {
+    this.$store.dispatch('notices/getNoticeById', { _id: this.noticeId });
+  },
+  data() {
+    return {
+      edit: false,
+      titleModel: this.title,
+      descriptionModel: this.description,
+      authorizedModel: ref(this.authorized),
+    };
+  },
+  methods: {
+    onEdit() {
+      this.edit = true;
+    },
+    onUpdate() {
+      const body = {
+        _id: this.noticeId,
+        title: this.titleModel,
+        description: this.descriptionModel,
+        authorized: this.authorizedModel,
+      };
+      this.$store.dispatch('notices/updateNotice', body).then(() => {
```

Comment:
Does it really need fetching the list of all notices from the backend in case updating the only one? Also, I'm not sure that updating the list will case re-fetching the information about this particular entity because the list of IDs probably wouldn't be changed.
I suppose it's better to do `dispatch('notices/getNoticeById', { _id: this.noticeId })` here